webpack.demo.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+const webpack = require('webpack');
+const path = require('path');
+
+const srcPath = path.join(__dirname, 'demo', 'src');
+const production = process.argv.indexOf('--dist') > -1;
+const outPath = production ? 'dist' : 'build';
+const devtool = production ? 'source-map' : 'eval-source-map';
+
+const config = {
```

Comment:
'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).